docs/cluster_configuration.md

```diff
@@ -177,3 +177,83 @@ status:
     policyNamespace: ztp-sno-ran-du-v4-16
     remediationAction: enforce
 ```
+
+### Updates to an existing ACM PolicyGenerator manifest
+
+For updating a manifest in an existing ACM PolicyGenerator, the following steps need to be taken (we'll take [sno-ran-du-pg-v4-16-v1](#./git-setup/policytemplates/version_4.Y.Z/sno-ran-du/sno-ran-du-v1.yaml) for examples):
+1. Upversion the cluster template content:
+    * Create a new version of the ACM PG: [sno-ran-du-pg-v4-16-v2](#./git-setup/policytemplates/version_4.Y.Z/sno-ran-du/sno-ran-du-v2.yaml):
+        * The name is updated to `sno-ran-du-pg-v4-16-v2` (the `ztp-sno-ran-du-v4-16` namespace is kept).
+        * `policyDefaults.labelSelector.sno-ran-du-policy` is updated from `v1` to `v2` such that the policy binding is updates.
+        * All policy names are updated from `v1` to `v2` (example: `v1-subscriptions-policy` -> `v2-subscriptions-policy`).
+        * The desired manifest section is updated. In the current samples [sno-ran-du-pg-v4-16-v2](#./git-setup/policytemplates/version_4.Y.Z/sno-ran-du/sno-ran-du-v2.yaml) adds a `sysctl` section to the `TunedPerformancePatch` section under the `v2-tuned-configuration-policy` policy.
+    * Create a new version of the [clusterinstance-defaults-v1](#./git-setup/clustertemplates/version_4.Y.Z/sno-ran-du/clusterinstance-defaults-v1.yaml): [clusterinstance-defaults-v2](#./git-setup/clustertemplates/version_4.Y.Z/sno-ran-du/clusterinstance-defaults-v2.yaml):
+        * Update the name to `clusterinstance-defaults-v2` (the namespace stays `sno-ran-du-v4-16`).
+        * Update the `sno-ran-du-policy` ManagedCluster extraLabel from `v1` to `v2`.
+    * Create a new version of the [sno-ran-du.v4-Y-Z-1](#./git-setup/clustertemplates/version_4.Y.Z/sno-ran-du/sno-ran-du-v1.yaml) `ClusterTemplate` CR: [sno-ran-du.v4-Y-Z-2](#./git-setup/clustertemplates/version_4.Y.Z/sno-ran-du/sno-ran-du-v2.yaml)
+        * Update the `metadata.name` from `sno-ran-du.v4-Y-Z-1` to `sno-ran-du.v4-Y-Z-2`
+        * Update `spec.version` from `v4-Y-Z-1` to `v4-Y-Z-2`
+        * Update `spec.templates.clusterInstanceDefaults` to `clusterinstance-defaults-v2`
+2. ArgoCD sync to the hub cluster:
+    * Add the newly created files to their corresponding kustomization.yaml.
+    * All the resources created from above are created on the hub cluster, including the `v2` policies and the new ClusterTemplate is validated.
+    * The new policies are not yet applied to the cluster because the `ManagedCluster` still has the old `sno-ran-du-policy: "v1"` label
+3. The SMO selects the new ClusterTemplate CR for the ProvisioningRequest:
+    * `spec.templateVersion` is updated from `sno-ran-du.v4-Y-Z-1` to `sno-ran-du.v4-Y-Z-2`
```

Comment:
Would it be:
spec.templateName = sno-ran-du
spec.templateVersion - 4-Y-Z-1 -> 4-Y-Z-2